DataFormats/HcalIsolatedTrack/src/classes_def.xml

```diff
@@ -36,4 +36,17 @@
   <class name="edm::Wrapper<reco::EcalIsolatedParticleCandidateCollection>"/>
   <class name="edm::reftobase::Holder<reco::Candidate, edm::Ref<std::vector<reco::EcalIsolatedParticleCandidate>,reco::EcalIsolatedParticleCandidate,edm::refhelper::FindUsingAdvance<std::vector<reco::EcalIsolatedParticleCandidate>,reco::EcalIsolatedParticleCandidate> > >" />
 
+  <class name="reco::HcalIsolatedTrackCandidate" ClassVersion="13">
+   <version ClassVersion="14" checksum="3313409542"/>
```

Comment:
The spurious future class version is a problem, please remove, it will hold the signature.